pegasus/routes/hoc_routes.rb

```diff
@@ -133,6 +133,7 @@
   end
 end
 
+# I think I'll need to modify this so the partner sites continue to go to old /congrats and code.org tutorials will go to new /congrats ?
```

Comment:
Looks like I can separate the contents of array and have one path for `partner_sites` and `csedweek.org` and another for `code.org` since we only want code.org tutorials to go to new congrats.

Reply:
What from the partner sites is going to the congrats page? A direct link?